### PR TITLE
Remove block relationships / add email to profile sink

### PR DESCRIPTION
**Pull request by bmtcril**
**Description:** 

*Breaking changes*
1- Removes the xblock relationships since they are expensive, unused, and difficult to work with. We should use the section/subsection/unit included on the blocks themselves.
2- Add the email address of the user to the user profile sink

**Testing instructions:**
1- You will need to add a column to your ClickHouse user profile table         
```
ALTER TABLE {{ ASPECTS_EVENT_SINK_DATABASE }}.{{ ASPECTS_EVENT_SINK_USER_PROFILE_TABLE }} 
        ADD COLUMN IF NOT EXISTS email String DEFAULT '' 
        AFTER name
``` 
2- Use this branch for your openedx Tutor containers, I usually just change `OPENEDX_EXTRA_PIP_REQUIREMENTS` to override the current version with `git+https://github.com/openedx/openedx-event-sink-clickhouse.git@bmtcril/remove_block_relationships`

3- `tutor images build openedx --no-cache`
4- Start tutor 
5- Test the email change by updating anything in your profile. You should see a new row in the user profile that has the correct email address.
5- Test the relationships by publishing a course. You should not see any new relationships rows in `event_sink.course_relationships`, but should see new dumps for `course_blocks` and `course_overviews`. Just check the `time_last_dumped` for each.

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** Updates will need to be made to Aspects Alembic migrations before it can be updated with this.

### CODE REVIEW

**Comment by Ian2012** [comment]
@bmtcril can you clean this folder too? https://github.com/openedx/openedx-event-sink-clickhouse/tree/main/event_sink_clickhouse

We should remove the `templates` folder and the `models.py` file.

File: test_utils/helpers.py [hunk 1]
```diff
@@ -342,46 +335,3 @@ def match(request):
...
-            for row in reader:
-                relation = relationships[i]
-                assert row[0] == relation[0]
-                assert row[1] == relation[1]
-                assert row[2] == relation[2]
-                i += 1
-        except AssertionError as e:
-            return False, f"Mismatch in row {i}: {e}"
-        return True, ""
-    return match
```
Comment (Ian2012) [inline test_utils/helpers.py:387]: why remove this test?

**Review by Ian2012 — COMMENTED** [review]
Just one question

File: test_utils/helpers.py (hunk 1 above)
Comment (bmtcril) [reply test_utils/helpers.py:387]: This was just a helper for testing the relationship CSVs, which we're not generating anymore.

**Review by Ian2012 — APPROVED** [review]
LGTM. Can you rename the model from `auth_user` to `retirement` or something more appropiate in the `RetirementSink`?

**Comment by bmtcril** [comment]
@Ian2012 I think that's a bigger change than I want to add to this, and I'm not honestly sure what the impact is. Can you create an issue and tag @pomegranited in it to see why it is the way it is?

**Comment by pomegranited** [comment]
@Ian2012 

>  Can you rename the model from `auth_user` to `retirement` or something more appropriate in the RetirementSink?

I think the model has to be `auth_user`, since that's the source model for the retirement signal?

**Comment by Ian2012** [comment]
@bmtcril Feel free to merge

**Pull request merged**